@types/interfaces.d.ts

```diff
@@ -1,12 +1,8 @@
-export type CustomFieldParseValueFn = (
-  value: any
-) => Array<any>;
+export type CustomFieldParseValueFn = (value: string) => Array<unknown>;
 
-export type CustomFieldFormatValueFn = (
-  inputValues: Array<any>
-) => any;
+export type CustomFieldFormatValueFn = (inputValues: Array<unknown>) => string;
 
-export type CustomFieldI18n = {
+export interface CustomFieldI18n {
```

Comment:
Just wondering: Do you have some guideline when we should use `interface` vs `type`? Quick googling tells me that the ways in which they can be used are almost identical so I'm not really sure why to prefer one over another, but would be good to try to keep some consistency. https://stackoverflow.com/a/52682220/1362634

Do you have any specific reason to use `type` for function signature and `interface` for object shape?

Reply:
Generally I tend to use `interface` for objects, but that's more like a personal preference.

Reply:
I was thinking that someone might want to augment the `i18n` interface, but wasn’t able to come up with a practical example so far.